### ISSUE TITLE
Create  a separate issue template for `Epic` issue

### ISSUE THREAD

**Issue by NadzeyaShpakouskaya** [ios-course/ironfoudation-team-project#126]
## Description

A new template for the Epic issue should contain next section:
 - Depends on tasks
 - Scope of the issue 
 - DoD (user story)
 - UI Mocks
 - Other helpful information

## Other helpful information
You can use this issue as an example:
https://github.com/ios-course/swiftcowboys-team-project/issues/45

**Comment by NadzeyaShpakouskaya** [comment]
Transferred to new project https://github.com/IronFoundation/iOS-Woof/issues/1
